src/scripts/main.js

```diff
@@ -1,3 +1,373 @@
+      <td>$${parseInt(salaryPerson).toLocaleString('en-EN')}</td>
+    </tr>`
+        );
+
+        pushNotification(450, 190, 'Success',
+          'The person was successfully added to the table', 'success');
+        break;
+    }
+  });
+}
+
+function deleteNotification() {
+  const notifications = document.querySelectorAll('.notification');
+
+  if (notifications.length > 0) {
+    notifications[0].remove();
+  }
+}
+
+function pushNotification(posTop, posRight, title, description, type) {
+  const titleMessage = document.createElement('h2');
+  const descriptionText = document.createElement('p');
+  const message = document.createElement('div');
+
+  message.style.position = 'fixed';
+  message.className = 'notification';
+  message.classList.add(type);
+  message.dataset.qa = 'notification';
+  titleMessage.className = 'title';
+
+  titleMessage.textContent = title;
+  descriptionText.textContent = description;
+  message.append(titleMessage);
+  message.append(descriptionText);
+  body.append(message);
+  message.style.top = `${posTop}px`;
+  message.style.right = `${posRight}px`;
+
+  setTimeout(() => {
+    message.remove();
+  }, 2000);
+};
+
+function replaceText() {
+  table.addEventListener('dblclick', (e) => {
+    e.preventDefault();
+
+    const item = e.target;
+
+    if (item.tagName !== 'TD' || !table.contains(item)) {
+      return;
+    }
+
+    editTd(item);
+  });
+
+  function editTd(td) {
+    const tdText = td.textContent;
+
+    const nameCell = td.cellIndex === 0;
+    const positionCell = td.cellIndex === 1;
+    const officeCell = td.cellIndex === 2;
+    const ageCell = td.cellIndex === 3;
+    const salaryCell = td.cellIndex === 4;
+
+    if (officeCell) {
+      const officePerson = body.querySelector('[name = "office"]');
+      const select = officePerson.cloneNode(true);
+
+      select.value = td.innerHTML;
+      td.innerHTML = '';
+      select.className = 'cell-input';
+      td.append(select);
+
+      select.focus();
+      savedText(select);
+    } else {
+      const input = document.createElement('input');
+
+      if (ageCell || salaryCell) {
+        input.type = 'number';
+      }
+
+      input.className = 'cell-input';
+      input.value = tdText;
+      td.innerHTML = '';
+      td.append(input);
+      input.focus();
+      savedText(input);
+    }
+
+    function savedText(tag) {
+      tag.addEventListener('blur', () => {
+        deleteNotification();
+        replaceValue(tag);
+      });
+
+      table.addEventListener('keypress', (e) => {
+        if (e.code === 'Enter') {
+          deleteNotification();
+          replaceValue(tag);
+        }
+      });
+    }
+
+    function replaceValue(tag) {
+      const emptyValue = tag.value.length === 0;
+
+      if (emptyValue) {
+        td.textContent = tdText;
+
+        pushNotification(450, 190, 'Error',
+          'The cell must not be empty!!! '
+          + 'The data are not changed!!!',
+          'error');
```

Comment:
closing bracket `)` should be on the same line as opening `(`, fix in other places